main.c

```diff
@@ -8,10 +8,30 @@
 
 static void activate (GtkApplication* app, gpointer user_data){
   GtkWidget *window;
+  GtkWidget *label;
+  GtkWidget *image;
+  GtkWidget *box;
+
   window = gtk_application_window_new (app);
+
+  box = gtk_box_new (GTK_ORIENTATION_VERTICAL, 5);
+
+  image = gtk_image_new_from_file("pumpkin.png");   
+
+  label = gtk_label_new("Hacktoberfest");
+
+
+  gtk_label_set_justify(GTK_LABEL(label), GTK_JUSTIFY_CENTER);
+
+  gtk_box_pack_start (GTK_BOX (box), label, TRUE, TRUE, 0);
+  gtk_box_pack_start (GTK_BOX (box), image, FALSE, FALSE, 0);
+
+  gtk_container_add(GTK_CONTAINER(window), box);
+
   gtk_window_set_title (GTK_WINDOW (window), "Window");
```

Comment:
Hi! Can you rename the window "Hacktoberfest 2019" ? :smile:

Reply:
For sure.  :)